### PR TITLE
Translations/missing (components)

### PR DESCRIPTION
**Pull request by vyvil**
PR to apply translations to several components that were previously missed.

### CODE REVIEW

File: src/components/ActivityFeed/ActivityFeedItem.js [hunk 1]
```diff
@@ -592,23 +599,23 @@ export class ActivityFeedItem extends React.Component<Props> {
           data = {
             ...data,
             subtext: t('label.outgoingSablierStream'),
-            itemValue: `- ${formattedAmount} ${symbol}`,
-            fullItemValue: `- ${formattedAmount} ${symbol}`,
+            itemValue: getValueWithSymbol(formattedAmount, symbol, false, false),
```
Comment (graszka22) [inline src/components/ActivityFeed/ActivityFeedItem.js:602]: Just a thought: wouldn't it be nicer to introduce functions like `getPositiveValueWithSymbol(amount, symbol)`? It'd be more readable than `getValueWithSymbol(formattedAmount, symbol, false, false)`, this way I have to wonder "what does this false, false mean?".

File: src/components/Selector/Selector.js [hunk 2]
```diff
@@ -181,7 +181,7 @@ const Selector = ({
   const hasValue = !isEmpty(selectedOption);
   const hasOptions = !!options?.length;
   const disabled = !hasOptions && !allowEnteringCustomAddress;
-  const placeholderText = !disabled ? `${placeholder}...` : 'no options to select';
+  const placeholderText = !disabled ? `${placeholder}...` : t('label.noOptionsToSelect');
```
Comment (graszka22) [inline src/components/Selector/Selector.js:184]: shouldn't the dots be somewhere in translation?

**Review by graszka22 — COMMENTED** [review]
LGTM 👍  two minor comments

File: src/components/ActivityFeed/ActivityFeedItem.js (hunk 1 above)
Comment (wjkg) [reply src/components/ActivityFeed/ActivityFeedItem.js:602]: I was going to mention the same thing, except with the alternative method of using an object to pass the options - so the parameter types would change from `(value: string, token: string, isPositive: boolean, doNotFormat: boolean)` to `(value: string, token: string, options: { isPositive: boolean, doNotFormat: boolean })`. This way one could still use conditions (like `formattedValue === '0'` in EventDetails).

Comment (vyvil) [reply src/components/ActivityFeed/ActivityFeedItem.js:602]: Yeah, good idea! Will update it shortly :) Thanks!

File: src/components/Selector/Selector.js (hunk 2 above)
Comment (vyvil) [reply src/components/Selector/Selector.js:184]: Oh, yeah. I'll add them as suffix. Thanks! :)

File: src/components/ActivityFeed/ActivityFeedItem.js (hunk 1 above)
Comment (vyvil) [reply src/components/ActivityFeed/ActivityFeedItem.js]: Renamed it too, to make it more readable :)

File: src/components/ActivityFeed/ActivityFeedItem.js [hunk 3]
```diff
@@ -268,7 +268,10 @@ export class ActivityFeedItem extends React.Component<Props> {
     const { amount, symbol, decimals }: AaveExtra = event.extra;
     if (!amount || !symbol) return '';
     const value = formatUnits(amount, decimals);
-    return getValueWithSymbol(formatAmount(value, getDecimalPlaces(symbol)), symbol, isPositive, !value);
+    return getFormattedValue(formatAmount(value, getDecimalPlaces(symbol)), symbol, {
+      isPositive,
+      doNotFormat: !value,
```
Comment (wjkg) [inline src/components/ActivityFeed/ActivityFeedItem.js:273]:
```suggestion
      noSymbol: !value,
```

Also, is it possible to catch this with type checking?

File: src/utils/strings.js [hunk 4]
```diff
@@ -20,14 +20,19 @@
...
 
-export const getValueWithSymbol =
-  (value: string, token: string, isPositive: boolean, doNotFormat: boolean) => {
-    if (doNotFormat) return t('tokenValue', { value, token });
-    if (isPositive) return t('positiveTokenValue', { value, token });
-    return t('negativeTokenValue', { value, token });
-  };
+export const getFormattedValue = (value: string, token: string, options: FormattedValueOptions = {}) => {
+  const { isPositive, noSymbol } = options;
+  if (noSymbol || isPositive === 'undefined') return t('tokenValue', { value, token });
```
Comment (wjkg) [inline src/utils/strings.js:35]: A missing `typeof`?

File: src/components/ActivityFeed/ActivityFeedItem.js (hunk 3 above)
Comment (vyvil) [reply src/components/ActivityFeed/ActivityFeedItem.js:273]: Uff ;/
Yeah. I'll add exact object type for this in a minute :)

File: src/utils/strings.js (hunk 4 above)
Comment (vyvil) [reply src/utils/strings.js:35]: Shoot, yup.
Fixing it and calling it a day :D
Thanks!

**Review by graszka22 — APPROVED** [review]

**Pull request merged**